sql/core/src/main/scala/org/apache/spark/sql/execution/python/TransformWithStateInPandasExec.scala

```diff
@@ -106,6 +106,30 @@ case class TransformWithStateInPandasExec(
     List.empty
   }
 
+    // operator specific metrics
+  override def customStatefulOperatorMetrics: Seq[StatefulOperatorCustomMetric] = {
```

Comment:
Shall we have some simple tests around custom metrics to ensure this works for python?